tablo/interfaces/arcgis/views.py

```diff
@@ -307,27 +340,83 @@ def generate_classified_renderer(classification_def, layer):
     }
 
 
-def covert_wkt_to_esri_feature(response_items):
+def convert_wkt_to_esri_feature(response_items, for_layer):
```

Comment:
Good catch: this method always has served to fill out the `features` key in the query response (though it gave special attention to geometry) so I never really considered the name except for the spelling correction.

How about `results_to_esri_features`?